config/locales/api/help/pa_Guru_IN.html

```diff
@@ -0,0 +1,313 @@
+            <h3>ਮਦਦ ਮੰਗੋ</h3>
+            <a href="%{help_screen_whatsapp_support_group_url}" class="whatsapp-button">%{icon_whatsapp_icon_svg} %{help_screen_whatsapp_support_number}</a>
+            <p>ਜੇ ਤੁਹਾਡੀਆਂ ਕੋਈ ਸਮੱਸਿਆਵਾਂ ਜਾਂ ਸਵਾਲ ਹਨ, ਕਿਰਪਾ ਕਰਕੇ ਸਾਡੇ WhatsApp ਗਰੁੱਪ ਲਈ ਬਟਨ ਦਬਾਓ। ਸਾਡੀ ਦੋਸਤਾਨਾ ਟੀਮ ਤੁਹਾਡੀ ਮਦਦ ਕਰੇਗੀ।</p>
+        </div>
+        
+        <p class="tos"><a href="https://simple.org/simple-privacy">ਗੋਪਨੀਯਤਾ ਨੀਤੀ</a> <span class="divider">&nbsp;|&nbsp;</span> <a href="https://simple.org/terms">ਸੇਵਾ ਦੀਆਂ ਸ਼ਰਤਾਂ</a></p>
+    </div>
+</div>
+    
+<div id="help-start" class="help-body" style="display: none;">
+    <div class="help-contents">
+<a href="#" onclick="close_window('help-start'); return false" class="help-back">%{icon_back_svg}</a><h2>ਸ਼ੁਰੂ ਕਰੋ</h2>
+        
+        <div class="card">
+            <h3>ਇੱਕ ਵੀਡੀਓ ਦੇਖੋ</h3>
+
+            <a href="%{help_screen_youtube_video_url}" class="video">%{thumb_demo_svg}</a>
+
+            <p>5 ਮਿੰਟਾਂ ਵਿੱਚ ਸਿੱਖੋ ਕਿ ਸਿੰਪਲ ਨੂੰ ਕਿਵੇਂ ਵਰਤਣਾ ਹੈ।</p>
+        </div>
+        
+        <div class="card">
+            <h3>ਸਿੰਪਲ ਵਿੱਚ ਕੌਣ ਰਿਕਾਰਡ ਕਰੇਗਾ?</h3>
+            <p>ਹਰੇਕ ਉਸ ਮਰੀਜ਼ ਬਾਰੇ ਰਿਕਾਰਡ ਰੱਖਣ ਲਈ ਸਿੰਪਲ ਦੀ ਵਰਤੋਂ ਕਰੋ <i>ਜਿਸ ਨੂੰ ਹਾਈਪਰਟੈਂਸ਼ਨ ਹੈ</i> ਜਾਂ ਜੋ ਹਾਈ ਬੀਪੀ ਦੀਆਂ ਦਵਾਈਆਂ ਲੈ ਰਿਹਾ ਹੈ।</p>
+            <p>ਜਦੋਂ ਕੋਈ ਮਰੀਜ਼ ਫੋਲੋ-ਅੱਪ ਲਈ ਵਾਪਸ ਆਉਂਦਾ ਹੈ, ਤਾਂ ਉਸ ਮਰੀਜ਼ ਨੂੰ ਲੱਭੋ ਅਤੇ ਨਵਾਂ ਬੀਪੀ ਦਾਖ਼ਲ ਕਰੋ।</p>
+        </div>
+        
+        <div class="card">
+            <h3>
+<span class="header-label"><span>ਸਟੈਪ 1</span></span>ਮਰੀਜ਼ ਦਾ ਬੀਪੀ ਲਓ</h3>
+            <p>ਬਿਲਕੁਲ ਸਹੀ ਬੀਪੀ ਲਓ: ਫਰਸ਼ ਉੱਤੇ ਪੈਰ, ਕੁਰਸੀ ਦੇ ਨਾਲ ਲੱਗੀ ਪਿੱਠ, ਬਿਲਕੁਲ ਵੀ ਗੱਲਬਾਤ ਨਾ ਕਰਨਾ।</p>
+        </div>
+        
+        <div class="card">
+            <h3>
+<span class="header-label"><span>ਸਟੈਪ 2</span></span>ਬੀਪੀ ਪਾਸਪੋਰਟ ਵਿੱਚ ਲਿਖੋ</h3>
+            <p>ਜੇ ਤੁਹਾਡੇ ਕੋਲ ਬੀਪੀ ਪਾਸਪੋਰਟ ਹਨ, ਤਾਂ ਮਰੀਜ਼ ਦਾ ਨਾਮ ਲਿਖੋ। ਫੇਰ, ਸਿੰਪਲ ਐਪ ਦੀ ਵਰਤੋਂ ਕਰਕੇ ਇਸਨੂੰ ਸਕੈਨ ਕਰੋ।</p>
+        </div>
+        
+        <div class="card">
+            <h3>
+<span class="header-label"><span>ਸਟੈਪ 3</span></span>ਮਰੀਜ਼ ਨੂੰ ਰਜਿਸਟਰ ਕਰੋ</h3>
+            <p>ਜੇ ਮਰੀਜ਼ ਨਵਾਂ ਹੈ, ਤਾਂ ਉਸ ਨੂੰ ਰਜਿਸਟਰ ਕਰੋ।</p>
+        </div>
+        
+        <div class="card">
+            <h3>
+<span class="header-label"><span>ਸਟੈਪ 4</span></span>ਸਿੰਪਲ ਵਿੱਚ ਬੀਪੀ ਦਾਖ਼ਲ ਕਰੋ</h3>
+            <p>ਬੀਪੀ ਰੀਡਿੰਗ ਨੂੰ <i>ਰਾਊਂਡ ਨਾ ਕਰੋ</i>।</p>
+        </div>
+        
+        <div class="card">
+            <h3>
+<span class="header-label"><span>ਸਟੈਪ 5</span></span>ਦਵਾਈਆਂ ਦਾਖ਼ਲ ਕਰੋ</h3>
+            <p>ਮਰੀਜ਼ ਦੀਆਂ ਵਰਤਮਾਨ ਹਾਈਪਰਟੈਂਸ਼ਨ ਦਵਾਈਆਂ ਦਾਖ਼ਲ ਕਰੋ।</p>
+        </div>
+        
+        <div class="card">
+            <h3>
+<span class="header-label"><span>ਸਟੈਪ 6</span></span>ਮਰੀਜ਼ ਨੂੰ ਸੁਰੱਖਿਅਤ ਕਰੋ</h3>
+            <p><span class="code">ਸੁਰੱਖਿਅਤ ਕਰੋ</span> ਨੂੰ ਦਬਾਓ ਅਤੇ ਮਰੀਜ਼ ਦੀ ਅਗਲੀ ਮੁਲਾਕਾਤ ਤੈਅ ਕਰੋ। ਬੀਪੀ ਪਾਸਪੋਰਟ &amp;apos;ਤੇ ਉਹ ਮਿਤੀ ਲਿਖੋ।</p>
```

Comment:
I'm not sure if this is your scope for this PR, but the original `pa_Guru_IN.html` here also has `&amp;` issues.